README.md

```diff
@@ -182,6 +165,21 @@ the app. To provide consistency we have a convention for these names:
   - **app-e2e**: to run the app with all the other apps necessary to provide
     full end to end user journeys.
 
+`govuk-docker startup` defaults to the `app` stack, but can be overridden in two ways:
+
+1. Any unnamed parameter will be taken as the stack name, with an `app-` prefix.
+2. Or you can specify a named `--stack` parameter.
+
+Example:
+
+```sh
+# Start content-publisher plus an "end-to-end" stack
+$ govuk-docker startup --service=content-publisher e2e
+
+# This command does the same thing as above
+$ govuk-docker startup --service=content-publisher --stack=app-e2e
```

Comment:
I think we want to discourage people using `startup` with a stack which isn't an `app` prefixed stack. Since `startup` is really just a thin wrapper around `run`, people should be using `run` in most situations.

I think ideally the command wouldn't even work if you pass `--stack`, it's more of an accident that it does due to the way we parse arguments across all commands in the same way.

Reply:
Removed the line